incubator/nfs-pv-provider/templates/nfs-server-pv.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: "{{ .Values.storage.name }}"
+  labels:
+    app: {{ template "nfs-pv-provider.name" . }}
+    chart: {{ template "nfs-pv-provider.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    demo: "{{ .Values.storage.demo_label }}"
```

Comment:
What's the demo label for?

Reply:
demo of how to add a label for use in the corresponding PVC as selector...

The README is going to be updated to show (three?) ways to select this PV: by storageClass, by name, and by label selector.  If this seems redundant, it may be.  I still haven't used this heavily, early WIP implementation.

Because of the nature of this PV, you really probably want to select it by name (so you get the one that has your data on it.)  I'm not really well versed on StatefulSet or multi-AZ schemes, but I have a couple of itches that scream "just use an NFS server" that needed to be scratched.

Reply:
imo, it's better to use PVCs where you can. Any reason not to do so here?

It's a frequently applied paradigm throughout most of these charts that when persistence is enabled, you either do dynamic provisioning by identifying a storage class _or_ (mutually exclusive) you identify an existing PVC.

It's not common to see PVs in these charts.

Reply:
Oh... sorry... I looked at the filename and assumed this was a PV and I didn't look any farther. This _is_ a PVC. Please fix the filename.

Reply:
Fixed the filename in 8baff259